minard/pmtnoisedb.py

```diff
@@ -17,12 +17,12 @@ def runs_after_run(run, maxrun=None):
     if maxrun:
         rows = conn.execute('SELECT * FROM pmtnoise'
                             ' WHERE %s < run_number AND run_number < %s'
-                            ' ORDER BY run_number DESC;', 
+                            ' ORDER BY run_number DESC LIMIT 500;', 
```

Comment:
Would be good to have this be a parameter. Usually for these functions, I've found it helpful to have a limit and offset parameter to the function which then get passed to the database with the following comand:

```python
rows = conn.execute('SELECT * FROM pmtnoise WHERE %s < run_number AND run_number < %s ORDER BY run_number DESC LIMIT %s OFFSET %s', run, maxrun, limit, offset);
```

Then on the page you can have next and back buttons to page through the results. You can have a look here: https://github.com/snoplus/minard/blob/7028e571572e94e13ae079bdd34c64d12c7b93d7/minard/views.py#L562 and https://github.com/snoplus/minard/blob/7028e571572e94e13ae079bdd34c64d12c7b93d7/minard/templates/nhit_monitor_thresholds.html#L136.